From: Tom Gundersen <teg@jklm.no>
Date: Wed, 30 Oct 2013 15:09:01 +0100
Subject: libudev: default log_priority to INFO

This brings it in sync with the default config file.
---
 src/libudev/libudev.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libudev/libudev.c b/src/libudev/libudev.c
index 208039a1..bc73a5a0 100644
--- a/src/libudev/libudev.c
+++ b/src/libudev/libudev.c
@@ -125,7 +125,7 @@ _public_ struct udev *udev_new(void)
                 return NULL;
         udev->refcount = 1;
         udev->log_fn = log_stderr;
-        udev->log_priority = LOG_ERR;
+        udev->log_priority = LOG_INFO;
         udev_list_init(udev, &udev->properties_list, true);
 
         f = fopen("/etc/udev/udev.conf", "re");
